.setup/bin/track_git_version.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
I'm against writing this to `submitty.json`. Given that we're not really using that info really anywhere meaningful, let's just write this to its own file `version.json` or something and that we don't care if it potentially gets blown away.